### ISSUE TITLE
RtForkTest.patchAndCheckJsonFork: Bind No free port within range

### ISSUE THREAD

**Issue by dpisarenko** [jcabi/jcabi-github#989]
A test, which I didn't touch fails during a Travis CI build.

https://travis-ci.org/jcabi/jcabi-github/jobs/47357190

```
RtForkTest.patchAndCheckJsonFork » Bind No free port within range: 39827=com.s...
```
Please make sure that the test runs.

**Comment by yegor256** [comment]
@dpisarenko this happens sometimes, in Travis environment. they don't have enough free TCP ports (which is rather strange). I agree that we should not crash in this case, but skip/ignore the tests gracefully.

**Comment by dmarkov** [comment]
this task will get someone's attention soon

**Comment by dmarkov** [comment]
@cyberone can you please help? Keep in mind [this](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html). If you have any technical questions, don't hesitate to ask right here. This task's budget is **30 mins**. This is exactly how much will be paid when the problem explained above is solved. See [this](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html) for more information

**Comment by dmarkov** [comment]
@dpisarenko thanks a lot for reporting, **15 mins** added to your acc, pmt ID `50570882`

**Comment by cyberone** [comment]
@dmarkov please review https://github.com/jcabi/jcabi-github/pull/996

**Comment by cyberone** [comment]
@dmarkov please do not complain on this delay. Give us more time, we're about to make a heavy decision.

**Comment by yegor256** [comment]
@dmarkov we're waiting for #996 here

**Comment by cyberone** [comment]
@dmarkov it's done. Please close the issue.

**Comment by dmarkov** [comment]
> @dmarkov please do not complain on this delay. Give us more time, we're about to make a heavy decision. 

@cyberone right, take your time and thanks for informing

**Comment by dmarkov** [comment]
> @dmarkov we're waiting for #996 here

@yegor256 issue #996 is closed already

**Comment by dmarkov** [comment]
once `989-33d6f233` puzzle is resolved (later, in another ticket), this ticket will be fully complete

**Comment by dmarkov** [comment]
@cyberone Much obliged! I have added **30 mins** to your account in payment "52066221". +30 added to your rating, current score is: [-610](http://www.netbout.com/b/35076?open=rating)

**Comment by 0pdd** [comment]
@dpisarenko all 2 puzzles are solved here: [#1017](https://github.com/jcabi/jcabi-github/issues/1017), [#1320](https://github.com/jcabi/jcabi-github/issues/1320).
